fdbserver/RestoreLoader.actor.cpp

```diff
@@ -882,3 +887,100 @@ ACTOR Future<Void> handleFinishVersionBatchRequest(RestoreVersionBatchRequest re
 	req.reply.send(RestoreCommonReply(self->id(), false));
 	return Void();
 }
+
+// Test splitMutation
+TEST_CASE("/FastRestore/RestoreLoader/splitMutation") {
```

Comment:
The test looks good, but also make me wonder if we should get rid of `splitMutation` function and just use the `KeyRangeMap` in the RestoreLoader code. In the end, reuse existing code is always better than come up with a new one.

Reply:
Yeah, that's the plan. 
First replace it and keep cross-validating the new one.
Then remove the splitMutation func.

Reply:
Sounds good. Maybe add a TODO in the code or issue so that you don't forget.